src/hooks/useComparison.ts

```diff
@@ -0,0 +1,47 @@
+import { useDispatch } from 'react-redux';
+
+import { RootState } from '../common/store';
+import { comparisonResults as secondRevisionResults } from '../mockData/9d5066525489';
+import { comparisonResults as thirdRevisionResults } from '../mockData/a998c42399a8';
+import { comparisonResults as firstRevisionResults } from '../mockData/bb6a5e451dac';
+import { setCompareData } from '../reducers/CompareResults';
+import { updateComparison } from '../reducers/ComparisonSlice';
+import { Strings } from '../resources/Strings';
+import { truncateHash } from '../utils/helpers';
+import { useAppSelector } from './app';
+
+const useComparison = () => {
+  const dispatch = useDispatch();
+  const comparisonResults = firstRevisionResults.concat(
+    secondRevisionResults,
+    thirdRevisionResults,
+  );
```

Comment:
This will be executed everytime `useComparison` is called. I'd suggest to do it in the last if block in `switchComparisonData` instead.